src/pages/workspace/tags/WorkspaceViewTagsPage.tsx

```diff
@@ -264,7 +276,9 @@ function WorkspaceViewTagsPage({route}: WorkspaceViewTagsProps) {
                         customListHeader={getCustomListHeader()}
                         shouldPreventDefaultFocusOnSelectRow={!DeviceCapabilities.canUseTouchScreen()}
                         listHeaderWrapperStyle={[styles.ph9, styles.pv3, styles.pb5]}
-                        onDismissError={(item) => Tag.clearPolicyTagErrors(policyID, item.value)}
+                        onDismissError={(item) => {
+                            Tag.clearPolicyTagErrors(policyID, item.value, item.orderWeight ?? 0);
```

Comment:
```suggestion
                            Tag.clearPolicyTagErrors(policyID, item.value, route.params.orderWeight);
```
Tags do not have `orderWeight` (see `tagList`)